src/raspberryEsp32Interface.cpp

```diff
@@ -148,14 +148,22 @@ namespace bzzz
     float RaspberryEsp32Interface::throttleReferencePercentage()
     {
         // Since PWM range is [1000, 2000], we subtract 1000 to calculate the percentage
-        //   percentage = (Referance_PWM - 1000)/1000
+        //   percentage = (Reference_PWM - 1000)/1000
         //=> percentage = Reference_PWM/1000 - 1
         return m_refData[RADIO_CHANNEL_THROTTLE] / 1000 - 1;
     }
 
+    bool RaspberryEsp32Interface::switchA()
+    {
+        return m_encodedSwitchesData & RADIO_SWITCH_A_BIT;
+    }
+    
     bool RaspberryEsp32Interface::armed()
```

Comment:
Think we should keep it `==` that means all the switches have to be in the correct place to arm

Reply:
Ah, yes! Maybe, let's put a comment there.

Reply:
I think the way it is now is the way it needs to be I've added comments

Reply:
@jamie-54, I renamed this to `canArm`, because it doesn't return whether the motors *are* armed, but whether they can be armed. The only place where we use this function is in the function `waitForArmCommand`:

```c++
void RaspberryEsp32Interface::waitForArmCommand()
{
    float temp[6];
    readPiData();
    sendFlightDataToPi(-1, -1, -1, -1, -1, -1, -1, -1, -1, -1);
    delay(20);
    while (!canArm())
    {
        readPiData();
        sendFlightDataToPi(-1, -1, -1, -1, -1, -1, -1, -1, -1, -1);
    }
}
```

If you're happy with this change we can merge this.

Reply:
Yea that makes sense thanks, I'm happy to merge